tests/seccomp.rs

```diff
@@ -0,0 +1,34 @@
+#[cfg(target_os = "linux")]
+use std::ffi::CString;
+
+#[cfg(target_os = "linux")]
+use birdcage::{Birdcage, Sandbox};
+
+#[cfg(target_os = "linux")]
+fn main() {
+    // Activate our sandbox.
+    Birdcage::new().lock().unwrap();
+
+    // Ensure `chdir` is allowed.
+    let root_path = CString::new("/").unwrap();
+    let result = unsafe { libc::chdir(root_path.as_ptr()) };
+    assert_eq!(result, 0);
+
+    // Ensure `unshare` is always blocked.
+    let result = unsafe { libc::unshare(libc::CLONE_NEWUSER) };
+    assert_eq!(result, -1);
+
+    // Ensure `clone` is blocked with `CLONE_NEWUSER`.
+    let stack = unsafe { libc::malloc(4096) };
+    let flags = libc::CLONE_NEWUSER as libc::c_ulong;
+    let result = unsafe { libc::syscall(libc::SYS_clone, flags, stack) };
```

Comment:
Why not use [`libc::clone`](https://docs.rs/libc/latest/libc/fn.clone.html)? The arguments are a bit more complicated (because you have to provide a function for the child), but it's probably less likely to cause weird effects...

Reply:
> The arguments are a bit more complicated

That's why. It's just calling `clone` underneath anyway.

Reply:
This way I can also ensure I'm calling clone, not clone3 with clone fallback.